webapp/graphql/Search.js

```diff
@@ -1,9 +1,15 @@
 import gql from 'graphql-tag'
-import { userFragment, postFragment, tagsCategoriesAndPinnedFragment } from './Fragments'
+import {
+  userFragment,
+  postFragment,
+  groupFragment,
+  tagsCategoriesAndPinnedFragment,
+} from './Fragments'
 
 export const searchQuery = gql`
   ${userFragment}
   ${postFragment}
+  ${groupFragment}
```

Comment:
Here we need `i18n` as parameter …

I think I've seen it somewhere als already.

Reply:
No